app/views/accounts/_application_table.html.erb

```diff
@@ -36,7 +36,7 @@
         </dd>
       </div>
 
-      <% if application.participant_outcome_state.present? && application.lead_provider_approval_status.capitalize == "Accepted" %>
+      <% if application.participant_outcome_state.present? && application.lead_provider_approval_status&.capitalize == "Accepted" %>
```

Comment:
Would be nice to have a nicer way of checking this, like
```
class Application
  def accepted?
    lead_provider_approval_status&.capitalize == "Accepted"
  end
end
```

Reply:
Yeah, nice catch. Updated the code likewise. 👍